src/Packages/Passport/Samples~/PassportManagerPrefab/README.md

```diff
@@ -0,0 +1,229 @@
+
+1. **Create UI**: Add Canvas → Button (Login), Button (Logout), Text (Status)
+2. **Drag Components**: In PassportUIController Inspector, drag the **COMPONENTS** (not GameObjects) to the UI fields:
+   - Expand the GameObject in hierarchy
+   - Drag the **Button** component (not the GameObject)
+   - For text: Drag **Text** component OR **TextMeshPro - Text (UI)** component (not the GameObject)
+   - PassportUIController supports both text types - use whichever your project uses
+3. **Configure**: Set your Client ID and redirect URIs
+4. **Done!**: Hit Play - your buttons now handle authentication automatically
+
+That's it! No scripts, no event handling, no code. Pure drag-and-drop! 🚀
+
+## 🎮 Using Events
+
+The prefab exposes Unity Events that you can wire up in the Inspector:
+
+### Initialization Events
+
+- `OnPassportInitialized`: Fired when Passport is ready
+- `OnPassportError`: Fired if initialization fails
+
+### Authentication Events
+
+- `OnLoginSucceeded`: User successfully logged in
+- `OnLoginFailed`: Login failed or was cancelled
+- `OnLogoutSucceeded`: User successfully logged out
+- `OnLogoutFailed`: Logout failed
+
+## 💻 Using from Code
+
+You can also interact with the PassportManager from your scripts:
+
+```csharp
+// Access the singleton instance
+var manager = PassportManager.Instance;
+
+// Check states
+if (manager.IsInitialized && manager.IsLoggedIn)
+{
+    // Access the underlying Passport instance
+    var email = await manager.PassportInstance.GetEmail();
+    var accessToken = await manager.PassportInstance.GetAccessToken();
+    
+    // Note: GetAddress() requires IMX connection via ConnectImx()
+    // var address = await manager.PassportInstance.GetAddress();
+}
+
+// Manual login with specific method
+manager.Login(DirectLoginMethod.Google);
+
+// Manual logout
+manager.Logout();
+```
+
+## 📁 Sample Scripts
+
+Check out the included example script:
+
+- `PassportUIController.cs`: Complete no-code UI integration (recommended)
+
+## ⚠️ Important: Cursor Management for Production
+
+The `PassportUIController` uses **aggressive cursor management** designed for demos and prototypes. It continuously unlocks the cursor to ensure UI remains clickable.
+
+### For Production Games
+
+1. **Disable `forceCursorAlwaysAvailable`** in PassportUIController Inspector
+2. **Implement custom cursor logic** based on your game's needs:
+   - **FPS games**: May want `Cursor.lockState = CursorLockMode.Locked` during gameplay
+   - **RTS games**: May want `Cursor.lockState = CursorLockMode.None` always
+   - **Menu-driven games**: May want cursor unlocked for UI, locked for gameplay
+3. **Handle cursor state** in the authentication event handlers:
+
+   ```csharp
+   void OnLoginSucceeded() {
+       // Your game-specific cursor behavior
+       Cursor.lockState = YourGame.GetDesiredCursorMode();
+   }
+   ```
+
+## 🔗 Deep Linking Setup
+
+For native platforms (Windows/Mac), you'll need to register your custom URL scheme:
```

Comment:
This section feels incomplete. Do we actually need it? The Passport docs already cover how to set up redirects, and on Windows the SDK handles it automatically, so no extra setup should be required.

Also, troubleshooting points like "Error loading user info: No IMX provider" don’t seem specific to this prefab. Would it make more sense to move these to the [Unity FAQ page](https://docs.immutable.com/build/unity/faq)? Then we could just add a note like “For more, check out the FAQ”.

Overall, I think this README could be much more concise. Right now there’s a lot to read. Less might be more here.